GetandTake/Core/Aspects/Caching/CacheAspect.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Castle.DynamicProxy;
+using GetandTake.Core.CrossCuttingConcerns.Caching;
+using GetandTake.Core.Utilities.Interceptors;
+using GetandTake.Core.Utilities.IoC;
+
+namespace GetandTake.Core.Aspects.Caching;
+
+public class CacheAspect : MethodInterception
+{
+    private const int defaultDuration = 60;
```

Comment:
Please, use _PascalCase_ for constants, it should be:
```suggestion
    private const int DefaultDuration = 60;
```